src/cnaas_nms/api/firmware.py

```diff
@@ -234,6 +234,13 @@ def post(self):
                 return empty_result(status='error',
                                     data='reboot should be a boolean')
 
+        if 'pre_flight' in json_data:
+            if isinstance(json_data['pre_flight'], bool):
+                kwargs['pre_flight'] = json_data['pre_flight']
+            else:
+                return empty_result(status='error',
+                                    data='reboot should be a boolean')
```

Comment:
The error message seems a bit off for pre_flight parameter?

Reply:
Fixed it.